server/app.js

```diff
@@ -93,9 +95,11 @@ async function start () {
     }
   })
 
+  // Add valid and beforeSave hooks here to ensure session is valid #TODO
```

Comment:
Related to #321

I'd like to see if we're not missing anything about session validation when we have an external validation service to trust.